ddtrace/utils/install.py

```diff
@@ -0,0 +1,24 @@
+from wrapt.importer import register_post_import_hook
+
+
+def install_module_import_hook(modulename, modulehook):
+    """Installs a module import hook for a given module.
+
+    A flag is also stored and checked on the module to so that the module is
+    not instrumented more than once.
+    """
+    # wrap the module hook with an idempotence check
+    def checked_hook(module):
+        if getattr(module, '_datadog_patch', False):
+            return
+        setattr(module, '_datadog_patch', True)
+        modulehook(module)
+
+    register_post_import_hook(checked_hook, modulename)
+
+
+def uninstall_module_import_hooks(modulename):
```

Comment:
I'm not sure if this is something we want to do or if it's even possible in `wrapt` (I don't see a public API for it).

Reply:
this could be a reason to have our own hook in `sys.meta_path` rather than using `@wrapt.when_imported()`

Reply:
The work-around I can see for now is to add a check like I mentioned in the above comment when we install the hook that will check to see if the integration is _enabled_ or not and will skip the patching if it's not enabled.